src/components/vm/vmRenameDialog.jsx

```diff
@@ -78,13 +88,25 @@ export const RenameDialog = ({ vmName, vmId, connectionName }) => {
             }}
             isHorizontal>
                 {!isObjectEmpty(error) && <ModalError dialogError={error.dialogError} dialogErrorDetail={error.dialogErrorDetail} />}
-                <FormGroup label={_("New name")}
+                <FormGroup label={_("Name")}
                            fieldId="rename-dialog-new-name">
                     <TextInput id='rename-dialog-new-name'
-                               validated={submitted && !newName ? "error" : "default"}
-                               value={newName}
-                               onChange={(_, value) => setNewName(value)} />
-                    <FormHelper helperTextInvalid={submitted && !newName && _("New name must not be empty")} />
+                               validated={submitted && !name ? "error" : "default"}
+                               value={name}
+                               onChange={(_, value) => setName(value)} />
+                    <FormHelper helperTextInvalid={submitted && !name && _("Name must not be empty")} />
+                </FormGroup>
+                <FormGroup label={_("Full name")}
```

Comment:
I wonder if this column can become dynamic? I.e. only show that if at least one VM actually *has* a full name? We haven't gotten any request about this for many years until recently, so I don't think a lot of users will bother with giving names/descriptions. 

It also shows in the updated pixel tests: they all have empty full names, and there isn't a new pixel test which demonstrates the name or description.

Reply:
> I wonder if this column can become dynamic? 

Yes, I was thinking the same.

Reply:
Done.